hydromt/models/model_plugins.py

```diff
@@ -156,8 +162,8 @@ def __str__(self):
         """Generate string representation containing the registered entrypoints."""
         plugins = "".join(
             [
-                f" - {name} ({self.eps[name].distro.name}"
-                f" {self.eps[name].distro.version})\n"
+                f" - {name} ({self.eps[name].dist.name}"
+                f" {self.eps[name].dist.version})\n"
```

Comment:
found and fixed a small issue here. The new EntryPoint class does not have a distro, but dist property. To test these lines I had to make some changes in the test as well as when loading the general hydromt model classes as entry points.